apps/license-api/src/app/modules/license/license.module.ts

```diff
@@ -0,0 +1,43 @@
+import { Module } from '@nestjs/common'
+import { LicenseController } from './license.controller'
+import { LicenseService } from './license.service'
+import { LOGGER_PROVIDER, logger } from '@island.is/logging'
+import { DisabilityLicenseApiClientModule } from './clients/disabilityLicense/disabilityLicenseClient.module'
+import { FirearmLicenseApiClientModule } from './clients/firearmLicense/firearmLicenseApiClient.module'
+import {
+  CLIENT_FACTORY,
+  GenericLicenseClient,
+  LicenseId,
+} from './license.types'
+import { DisabilityLicenseClientService } from './clients/disabilityLicense/disabilityLicenseClient.service'
+import { FirearmLicenseApiClientService } from './clients/firearmLicense/firearmLicenseApiClient.service'
+
+@Module({
+  imports: [DisabilityLicenseApiClientModule, FirearmLicenseApiClientModule],
```

Comment:
Rest resources should respond with Problem JSON on [errors](https://docs.devland.is/technical-overview/api-design-guide/errors) and we have the [`ProblemModule`](https://docs.devland.is/libs/nest/problem) which take care of formatting thrown exceptions as Problem JSON.

Reply:
We import the ProblemModule in the app root, isn't that sufficient?

Reply:
Ahh of course, didn´t notice that. Yes that is sufficient.